packages/client/TODO.md

```diff
@@ -93,11 +93,11 @@ Client manages the token lifecycle and storage
 
 ### Query publications
 
-- [x] [Q] client.publication.fetch(request) or client.publications.fetchOne()
+- [x] [Q] client.publication.fetch(request)
 - [x] [PQ] client.publication.fetchAll(request)
 - [x] [Q] client.publication.validateMetadata(request)
-- [x] [PQ] client.publication.allWhoCollected({ publicationId })
-- [x] [PQ] client.publication(s).allForSale({ profileId })
+- [x] [PQ] client.publication.allWalletsWhoCollected({ publicationId })
```

Comment:
the returned items are an array of Wallets, not Publications like in other methods in this module